### ISSUE TITLE
All small buttons should have a height of 24 (minimum click-target for a11y)

### ISSUE THREAD

**Issue by jo-oikawa** [microsoft/vscode#267862]
Today the 'small' button size has a height of 22px with 11px left- and right-padding. 
To meet minimum click target size requirements, we need the button to be 24px high. 
I believe we can keep the size of the font and other elements within the button the same, and just have a slightly bigger top- and bottom-padding.
